edgedb-client/src/credentials.rs

```diff
@@ -19,7 +19,9 @@ pub struct Credentials {
     #[serde(default, skip_serializing_if="Option::is_none")]
     pub tls_cert_data: Option<String>,
     #[serde(default, skip_serializing_if="Option::is_none")]
-    pub tls_verify_hostname: Option<bool>,
+    pub tls_verify_hostname: Option<bool>, // deprecated
```

Comment:
I think this should be private a private field already. So we don't introduce breaking changes when we remove it.